src/tests/Navbar.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import Navbar from '../Components/Navbar';
+import store from '../redux/configStore';
+
+describe('Navbar tests', () => {
+  test('should render', () => {
+    expect(Navbar).toMatchSnapshot();
+  });
+
+  test('Checks if header content is there by checking the logo text', () => {
+    render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Navbar />
+        </BrowserRouter>
+        ,
+      </Provider>,
+    );
+
+    expect(
+      screen.getByRole('heading', { name: /COVID METRICS/i }),
+    ).toBeInTheDocument();
+  });
```

Comment:
- [ ] Make sure that all your test cases are correct. For now, this test failed, seems that the expectation is wrong. Kindly solve this issue by changing the name with the right input. ( It should be "2022" for your case).
![image](https://user-images.githubusercontent.com/71874425/178022154-8ebd57c5-903d-4840-9bf0-3e657db85f61.png)